lib/commons/color/get-background-color.js

```diff
@@ -118,6 +118,15 @@ color.filteredRectStack = function filteredRectStack(elm) {
 		const boundingStack = rectStack.shift();
 		let isSame;
 
+		// Safari v12.1 does not include labels as part of elementsFromPoint()
+		// if they wrap an input element (UNLESS the label has a background
+		// color). this results in two different rectStacks (since
+		// elm.getClientRects() returns two rects for the element) which
+		// returns null as isSame is false. we can fix this by adding in the
+		// missing label to the boundingStack before checking for isSame
+		// @see https://bugs.webkit.org/show_bug.cgi?id=197743
+		includeMissingElements(boundingStack, elm);
```

Comment:
This is already happening in `getBackgroundStack`. I don't quite understand why you'd call this twice.

Reply:
So the `includeMissingElements` call in `getBackgroundStack` is not called since `filteredRectStack` returns `null` because it's missing the label. I tried removing the `includeMissingElements` in `getBackgroundStack` but that failed a lot of tests.